build.gradle

```diff
@@ -39,7 +39,7 @@ allprojects {
   ext {
     scalaBinaryVersion = '2.11'
     scalaVersion = scalaBinaryVersion + '.8'
-    snappySparkVersion = '2.0.2.3'
+    sparkVersion = '2.0.0'
```

Comment:
snappySparkVersion is no longer used (after the removal of snappy-spark-unsafe dependency). Upstream spark is used for spark-unsafe which currently gets inherited from snappydata but ideally store should be independently buildable too.